Assets/Scripts/CameraFollow.cs

```diff
@@ -15,57 +15,64 @@ public class CameraFollow : MonoBehaviour
     public float bottom = -5f;
     public float top = 5f;
 
-	private Vector3 lerpedPosition;
-
+    private Vector3 _lerpedPosition;
```

Comment:
> was war der grund für die umbennenung? und wieso mit underline? wolltest du das alle privaten felder mit unterstrich sind? Brauchen wir dafür eine Coding Guidline? in anderen Teilen des projekts ist es noch nicht so umgesetzt. wir könnten den Microsoft C# guidelines folgen.. https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions
> 
> finde es allgemein komisch variablen gross zu schreiben, aber wen das die Guideline ist, denke kann ich mich auch umgewöhnen.
Underline was Rider default. I assumed this will be our Guidelines
I also find it awkward to capitalize vars, but is C# standard. While lower case vars seem to be unity standard. I tried to follow unity standard here. Also lowercase were not marked by IDE but dont know if Uppercase Vars will.

Reply:
> oder wir machen es so: https://avangarde-software.com/unity-coding-guidelines-basic-best-practices/

We should probably define which guidelines to use in the next meeting
preferably use some with coding guidelines files to import in all ides used by the team